include/dtpcontrols/AllNodeNames.hpp

```diff
@@ -0,0 +1,150 @@
+/**
+ * @file AllNodeNames.hpp
+ *
+ * THIS IS A AUTO-GENERATED FILE.
+ * PLEASE DO NOT EDIT.
+ *
+ * AllNodeNames is a namespace which includes all
+ * unique ipbus node names within DTP-firmware
+ * which is part of DUNE DAQ.
+ *
+ * This is part of the DUNE DAQ Software Suite, copyright 2022.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+
+#ifndef DTP_INCLUDE_DTP_ALLNODENAMES_HPP_
```

Comment:
Include guards doesn't follow DUNE-DAQ conventions.

Reply:
Ok, shall fix this too.